packages/notifi-react-card/lib/context/NotifiSubscriptionContext.tsx

```diff
@@ -79,6 +89,15 @@ export const NotifiSubscriptionContextProvider: React.FC<
   const [useHardwareWallet, setUseHardwareWallet] = useState<boolean>(false);
   const [loading, setLoading] = useState(false);
 
+  const alertConfigurations = useRef<Record<string, AlertConfiguration | null>>(
+    params.alertConfigurations ?? {},
+  );
+  const getAlertConfigurations = useCallback((): Readonly<
+    Record<string, AlertConfiguration | null>
+  > => {
+    return alertConfigurations.current;
+  }, []);
+
```

Comment:
how does this fix the issue? nobody actually sets the data, do they?

Reply:
hmm, thought again and I think this fix approach is not correct, updated to use targetGroupName to get the correct targetGroup